src/test/model-viewer-base-spec.ts

```diff
@@ -211,66 +211,76 @@ suite('ModelViewerElementBase', () => {
 
       suite('toBlob', () => {
         test('produces a blob', async () => {
-          const blob = await element.toBlob();
+          const blob = await element.toBlob({});
           expect(blob).to.not.be.null;
         });
 
         test('can convert blob to object URL', async () => {
-          const blob = await element.toBlob();
+          const blob = await element.toBlob({});
           const objectUrl = URL.createObjectURL(blob);
           const objectUrlMatcher = /^blob\:/;
           expect(objectUrlMatcher.test(objectUrl)).to.be.true;
         });
 
         test('has size', async () => {
-          const blob = await element.toBlob();
+          const blob = await element.toBlob({});
           expect(blob.size).to.be.greaterThan(0);
         });
 
         test('uses fallbacks on unsupported browsers', async () => {
           // Emulate unsupported browser
           let restoreCanvasToBlob = () => {};
           try {
-            restoreCanvasToBlob = spy(HTMLCanvasElement.prototype, 'toBlob', { value: undefined });
+            restoreCanvasToBlob =
+                spy(HTMLCanvasElement.prototype, 'toBlob', {value: undefined});
           } catch (error) {
             // Ignored...
           }
 
-          const blob = await element.toBlob();
+          const blob = await element.toBlob({});
           expect(blob).to.not.be.null;
 
           restoreCanvasToBlob();
         });
 
-        test('blobs on supported and unsupported browsers are equivalent', async () => {
-          // Skip test on IE11 since it doesn't have Response to fetch arrayBuffer
-          if (IS_IE11) {
-            return;
-          }
-
-          let restoreCanvasToBlob = () => {};
-          try {
-            restoreCanvasToBlob = spy(HTMLCanvasElement.prototype, 'toBlob', { value: undefined });
-          } catch (error) {
-            // Ignored...
-          }
-
-          const unsupportedBrowserBlob = await element.toBlob();
-
-          restoreCanvasToBlob();
-
-          const supportedBrowserBlob = await element.toBlob();
-
-          // Blob.prototype.arrayBuffer is not available in Edge / Safari
-          // Using Response to get arrayBuffer instead
-          const supportedBrowserResponse = new Response(supportedBrowserBlob);
-          const unsupportedBrowserResponse = new Response(unsupportedBrowserBlob);
-
-          const supportedBrowserArrayBuffer = await supportedBrowserResponse.arrayBuffer();
-          const unsupportedBrowserArrayBuffer = await unsupportedBrowserResponse.arrayBuffer();
-
-          expect(unsupportedBrowserArrayBuffer).to.eql(supportedBrowserArrayBuffer);
-        });
+        test(
+            'blobs on supported and unsupported browsers are equivalent',
```

Comment:
It will require a little setup, but it would be nice to see a test added for `idealAspect`. I think it could roughly look like:

 1. Load a model with `<model-viewer>`
 2. Get `Blob`s with and without `idealAspect`
 3. Create object URLs for each blob
 4. Create `img` tags and assign object URLs as `img.src`
 5. Wait for images to load
 6. Compare the `width` and `height` of the images to expectations

Reply:
Done.